docs/application/web/api/5.5/ui_fw_api/Mobile_UIComponents/mobile_Coverflow.htm

```diff
@@ -0,0 +1,233 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN">
+<html>
+<head>
+	<script type="text/javascript" src="../snippet.js"></script>
+	<meta http-equiv="content-type" content="text/html; charset=utf-8">
+	<link href="../styles.css" rel="StyleSheet" type="text/css">
+	<link href="../snippet.css" rel="StyleSheet" type="text/css">
+	<title>Coverflow</title>
+</head>
+<body onload="prettyPrint()" id="content">
+
+
+<h1>Coverflow</h1>
+
+<p>The coverflow component visually applies various effects to selected images.</p>
+
+<img src="../../images/mobile_coverflow.png" width="10%">
+
+<h2>Table of Contents</h2>
+
+<ol class="toc">
+
+		<li><a href="#default-selectors-coverflow-mobile">Default Selectors</a></li>
+
+		<li><a href="#html-examples-coverflow-mobile">HTML Examples</a></li>
+
+		<li><a href="#options-for widget-coverflow-mobile">Options</a></li>
+
+		<li><a href="#methods-coverflow-mobile">Methods</a></li>
+
+</ol>
+
+<h2><a id="default-selectors-coverflow-mobile"></a>Default Selectors</h2>
+
+<p>Elements with the <code>class=&quot;ui-coverflow&quot;</code> attribute can create a Coverflow.<br>
+Changing the value of <code>data-effect=&quot;coverflow&quot;</code> applies a variety of effects.<br>
+Coverflow images are defined inside <code>&lt;ul class=&quot;flip-items&quot;&gt;</code>.<br>
```

Comment:
Coverflow images are defined inside <code>&lt;ul class=&quot;flip-items&quot;&gt;</code>.
Images are recognized as coverflow when declared with the <code>&lt;li&gt;</code> tag.</p>

How about below? Please check if the sentence is correct.

Coverflow images are defined inside <code>&lt;ul class=&quot;flip-items&quot;&gt;</code>, declared with the <code>&lt;li&gt;</code> tag.

Reply:
good. I applied.

done.